Ogury/src/main/java/com/mopub/mobileads/OguryInitializer.java

```diff
@@ -23,6 +25,8 @@
     private static final String MONITORING_KEY_MODULE_VERSION = "mopub_ce_version";
     private static final String MONITORING_KEY_MOPUB_VERSION = "mopub_mediation_version";
 
+    private static final String CHOICE_MANAGER_CONSENT_ORIGIN = "MoPub";
```

Comment:
According to the spec, it should be "MOPUB"

Reply:
Fixed in f06768509dc6cb8cd9304a8b6f0272eec00defc1